packages/react/README.md

```diff
@@ -5,13 +5,579 @@
   <h1 align="center">Sanity App SDK (React)</h1>
 </p>
 
-React hooks for creating Sanity applications.
+React hooks for creating Sanity applications. Live by default, optimistic updates, multi-project support.
+
+---
+
+## Quickstart
+
+### 1. Setup (2 min)
+
+```bash
+npx sanity@latest init --template app-quickstart
+cd your-app
+npm run dev
+```
+
+Opens at `https://www.sanity.io/welcome?dev=http%3A%2F%2Flocalhost%3A3333`, proxied through Sanity Dashboard for auth.
+
+**Key files:**
+
+- `sanity.cli.ts` — CLI config, app entry point
+- `src/App.tsx` — Root with `<SanityApp>` provider
+- `src/ExampleComponent.tsx` — Your starting point
+
+### 2. Configuration
+
+```tsx
+import {SanityApp, type SanityConfig} from '@sanity/sdk-react'
+
+const config: SanityConfig[] = [
+  {projectId: 'abc123', dataset: 'production'},
+  {projectId: 'def456', dataset: 'production'}, // multi-project support
+]
+
+export function App() {
+  return (
+    <SanityApp config={config} fallback={<div>Loading...</div>}>
+      <YourApp />
+    </SanityApp>
+  )
+}
+```
+
+**Auth is automatic** — Dashboard injects token via iframe. No login flow needed.
+
+---
+
+## Guide
+
+### Document Handles
+
+As a core concept, everything revolves around handles — minimal pointers to documents:
+
+```tsx
+type DocumentHandle = {
+  documentId: string
+  documentType: string
+  projectId?: string // optional if using the default projectId or inside a ResourceProvider
+  dataset?: string // optional if using the default dataset or inside a ResourceProvider
+}
+```
+
+**Pattern:** Fetch handles first → pass to child components → fetch content there.
+
+---
+
+### Hook Reference
+
+#### Data Retrieval
+
+```tsx
+// Collection of handles (infinite scroll)
+const {data, hasMore, loadMore, isPending, count} = useDocuments({
+  documentType: 'article',
+  batchSize: 20,
+  orderings: [{field: '_updatedAt', direction: 'desc'}],
+  filter: 'status == "published"', // GROQ filter
+  params: {status: 'published'}, // filter params
```

Comment:
Thinking something like this might demonstrate the combination of `filter` and `params` a bit more:

```suggestion
  filter: 'status == $status', // GROQ filter
  params: {status: 'published'}, // Parameters used for the GROQ filter
```